schema/transportation/segment.yaml

```diff
@@ -51,6 +57,71 @@ properties:
         default: []
 "$defs":
   propertyDefinitions:
+    destinationLabelType:
+      description: >-
+        The type of object of the destination label.
+      type: string
+      enum: 
+      - place
+      - street
+      - country 
+      - route
+    destinations:
+      description: >-
+        Information about what destinations can be reached if following this segment. 
+        Includes references to the segments from which it is needed to transition 
+        onto this segment to reach the destinations.
+      type: array
+      items:
+        type: object
+        unevaluatedProperties: false
+        required: [labels, from, to_connector_id, final_heading]
+        properties:
+          labels:
+            description: >-
+              Describes the destination labels on the directions sign.
```

Comment:
nit: What if you simplified this to "Labeled destinations that can be reached by following the segment"?

Reply:
done